permissions/plugin-graphql-server.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "graphql-plugin"
+github: "jenkinsci/graphql-plugin"
+paths:
+- "org/jenkins-ci/plugins/graphql-plugin"
```

Comment:
This path is incorrect, it should match the groupId/artifactId from your pom.xml (https://github.com/jenkinsci/graphql-server-plugin/blob/master/pom.xml#L9). So, you would need "io/jenkins/plugins/graphql-server"

Reply:
thanks, skimmed it too fast.